docs/setup/install.md

```diff
@@ -1,16 +1,57 @@
 # Quick setup guide
 
-Before going further, make sure you have ROS 2 (Jazzy or Humble) installed and sourced on your
-system. If you don't have ROS 2 follow the installation documentation for
-[Humble](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debs.html) or
-[Jazzy](https://docs.ros.org/en/jazzy/Installation/Ubuntu-Install-Debs.html). Make sure that
-`ros-dev-tools` are installed.
+Before going further, make sure you have ROS 2 (jazzy or humble) installed and sourced on your system.
 
 !!! tip "Docker images"
 
     RAI has experimental docker images. See the [docker](setup_docker.md) for
     instructions.
 
+There are two ways to start using RAI:
+
+1. Installing RAI using pip (recommended for end users)
+
+2. Setting up a developer environment using poetry (recommended for developers)
+
+## Installing RAI
+
+1.  Install core functionality:
+
+    ```bash
+    pip install rai-core
+    ```
+
+2.  Initialize the global configuration file:
+
+    ```bash
+    rai-config-init
+    ```
+
+3.  Optionally install ROS 2 dependencies:
+
+    ```bash
+    sudo apt install ros-${ROS_DISTRO}-rai-interfaces
+    ```
+
+!!! important "Package availability"
+
+    `rai_openset` and `rai_nomad` are not yet available through pip. If your workflow relies on openset detection or NoMaD integration, please refer to the
+    [developer environment instructions](#setting-up-developer-environment) setup.
+
+    `rai_interfaces` is available through rosdep. However, due to package distribution delays, the latest version may not be immediately available. If you encounter missing imports, please build `rai_interfaces` from [source](https://github.com/RobotecAI/rai_interfaces).
```

Comment:
how to "build from source" might not be clear. I would document this process. (clone to src -> colcon build -> source workspace)